processor/blockprocessor/block_processor.go

```diff
@@ -2,6 +2,7 @@ package blockprocessor
 
 import (
 	"fmt"
+
```

Comment:
TODO: Curerently it appears that there are no changes of substance in this file. Confirm, and remove this change before merging.